play-scala-secure-session-example/app/Module.scala

```diff
@@ -1,10 +1,19 @@
+import akka.actor.ActorSystem
+import akka.actor.typed.Scheduler
 import com.google.inject.AbstractModule
+import javax.inject.{ Inject, Provider, Singleton }
 import play.api.libs.concurrent.AkkaGuiceSupport
 import services.session.{ ClusterSystem, SessionCache }
 
 class Module extends AbstractModule with AkkaGuiceSupport {
   override def configure(): Unit = {
     bind(classOf[ClusterSystem]).asEagerSingleton()
-    bindActor[SessionCache]("replicatedCache")
+    bindTypedActor(SessionCache.create(), "replicatedCache")
+    bind(classOf[Scheduler]).toProvider(classOf[AkkaSchedulerProvider])
   }
 }
+
+@Singleton class AkkaSchedulerProvider @Inject() (system: ActorSystem) extends Provider[Scheduler] {
+  import akka.actor.typed.scaladsl.adapter._
+  override lazy val get: Scheduler = system.toTyped.scheduler
+}
```

Comment:
This won't be necessary after https://github.com/playframework/playframework/pull/9758. Let's add a reference to the PR.